src/Tribe/PUE/Checker.php

```diff
@@ -780,23 +780,58 @@ public function maybe_display_json_error_on_plugins_page( $page ) {
 			}
 
 			$state = $this->get_option( $this->pue_option_name, false, false );
+			$messages = array();
+			$plugin_updates = get_plugin_updates();
+			$update_available = isset( $plugin_updates[ $this->plugin_file ] );
+
+			// Check to see if there is an licensing error or update message we should show
+			if ( ! empty( $state->update->license_error ) ) {
+				$messages[] = $state->update->license_error;
+			} elseif ( $update_available && current_user_can( 'update_plugins' ) ) {
+				// A plugin update is available
+				$update_now = sprintf(
+					esc_html__( 'Update now to version %s.', 'tribe-common' ),
+					$state->update->version
+				);
 
-			if ( empty( $state->update->license_error ) ) {
-				return;
+				$update_now_link = sprintf(
+					' <a href="%1$s" class="update-link">%2$s</a>',
+					wp_nonce_url( self_admin_url( 'update.php?action=upgrade-plugin&plugin=' ) . $this->plugin_file, 'upgrade-plugin_' . $this->plugin_file ),
+					$update_now
+				);
+
+				$update_message = sprintf(
+					esc_html__( 'There is a new version of %1$s available. %2$s', 'tribe-common' ),
+					$this->plugin_name,
+					$update_now_link
+				);
+
+				$messages[] = sprintf(
+					'<p>%s</p>',
+					$update_message
+				);
 			}
 
-			$this->plugin_notice = array(
-				'slug' => $this->get_slug(),
-				'message_row_html' => "
-					<tr class='plugin-update-tr active'> <td colspan='3' class='plugin-update'>
-						<div class='update-message notice inline notice-warning notice-alt'>
-							{$state->update->license_error}
-						</div>
-					</td> </tr>
-				",
-			);
+			if ( ! empty( $messages ) ) {
```

Comment:
I'd love to see this changed to:

```
if ( empty( $messages ) ) {
  return;
}
```

That way the meat of what we are expecting to have happen, isn't nested in.